### PR TITLE
Celigo util

### PR DESCRIPTION
**Pull request by hughes036**
This pr adds a util module with (unit tested) methods that the pipeline and [data fix script(s)](https://aicsbitbucket.corp.alleninstitute.org/projects/SW/repos/labkey-db-fixes/browse/bin/2022/2022_12_8-update_create_celigo_annotations.py) can share.

### CODE REVIEW

File: aics_pipeline_uploaders/util/test/test_celigo.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import json
+import requests
```
Comment (BrianWhitneyAI) [inline aics_pipeline_uploaders/util/test/test_celigo.py]: requests is unused in this file.

File: aics_pipeline_uploaders/util/celigo.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+class CeligoUtil():
+
+    def __init__(self, env: str = "stg"):
+        self.env = env
+
+    def lookup_well_id(self, plate_barcode, well_name):
+        well_info_response = requests.get(ENV_SERVICE_MAP[self.env][SERVICE_MMS][MMS_RESOURCE_PLATE_INFO] + str(plate_barcode))
+        well_info = well_info_response.json()
+        well_name_lookup = well_info['data'][0]['wellNameLookup']
```
Comment (BrianWhitneyAI) [inline aics_pipeline_uploaders/util/celigo.py:33]: We had talked about adding some handling here where multiple imaging sessions had occured. Currently this method just takes the first element of that returned well_info['data'] list.

**Review by BrianWhitneyAI — COMMENTED** [review]
This project is covered under the **Allen Institute Software License**.

Looks much better than what I had before. A couple things I noticed:

1) We still have an issue with the lookup_well_id returning just the first entry of well_info.
2) requests is not used in test_celigo.py

Additionally, In the past I had used a set of dev tools for formatting (black, isort, flake8, mypy). When checking this branch against those standards there are some minor organizational changes we may want to make.

File: aics_pipeline_uploaders/util/celigo.py (hunk 2 above)
Comment (hughes036) [reply aics_pipeline_uploaders/util/celigo.py:33]: Added handling for this.

File: aics_pipeline_uploaders/util/test/test_celigo.py (hunk 1 above)
Comment (hughes036) [reply aics_pipeline_uploaders/util/test/test_celigo.py]: Removed.

**Review by BrianWhitneyAI — APPROVED** [review]
lgtm

**Pull request merged**